test/integration/render-tests/image/raster-scaling/style.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 512,
```

Comment:
We like to try to keep render tests to 64 x 64px to minimize the time it takes to run the test suite — as long as linear vs nearest results can still be clearly seen at smaller sizes, please reduce the size of these new render tests.

Reply:
No problem, I've changed this to 64x64, I used 512 simply because all the other tests use this.